loops-and-arrays.js

```diff
@@ -53,3 +63,11 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+for (let i = 0; i < squares.length; i++) {
+    let volume = squares[i] * squares[i] * squares[i];
```

Comment:
Const had hier ook prima gemogen, aangezien het niet opnieuw overschreven wordt in dezelfde sessie